cmasher/utils.py

```diff
@@ -17,13 +15,19 @@
 from typing import TYPE_CHECKING, Callable, Optional, Union
 
 import matplotlib as mpl
+import matplotlib.pyplot as plt
```

Comment:
We avoid importing `pyplot` at the top level of this module `cmasher.utils` is run with `import cmasher` but `pyplot` is *not* needed in the most common usage of cmasher, which is to simply define colormaps, but `pyplot` has a significant impact on startup time, and we want `import cmasher` to take as little time as possible in case anyone imports cmasher in a CLI.

Reply:
Thanks for the explaination @neutrinoceros. I would move it back to where it was. Deeply appreciate it!